synapse/http/server.py

```diff
@@ -342,7 +350,7 @@ def _send_response(self, request, code, response_json_object,
 
 
 def _options_handler(request):
-    return {}
+    return 200, {}
```

Comment:
I was just finding it very confusing to try to guess whether the {} is a response body, or parsed URL segments, or what - especially _get_handler_for_request seems to append its own {} tuple alongside the returned handler... just a snidget of pydoc feels like it would be useful for confused bears

Reply:
(especially as it looks like you were confused too, based on missing the 200)